src/diff/index.js

```diff
@@ -215,6 +224,12 @@ export function diff(
 
 		if ((tmp = options.diffed)) tmp(newVNode);
 	} catch (e) {
+		if (isHydrating) {
+			// Before bailing out, mark the current VNode with the DOM element and hydration state.
+			// We can use this information if we return here to render later on.
+			oldVNode._hydrateDom = newVNode._dom = oldDom;
+			oldVNode._hydrating = true;
```

Comment:
I think if we're merging this with the `if(isHydrating)` condition, we should drop the `_hydrating` prop. It's always `true` when `vnode._hydrateDom != null`.

Reply:
done